sharpy/client.py

```diff
@@ -102,11 +102,10 @@ def make_request(self, path, params=None, data=None, method=None):
             cleaned_data = data.copy()
             try:
                 del cleaned_data['subscription[ccCardCode]']
+                ccNum = cleaned_data['subscription[ccNumber]']
+                cleaned_data['subscription[ccNumber]'] = ccNum[-4:]
```

Comment:
It is generally bad form to use exceptions for flow control. You should instead do:

    if 'subscription[ccNumber]' in cleaned_data:
        cleaned_data['subscription[ccNumber]'] = cleaned_data[subscription[ccNumber]][:-4]